.gitignore

```diff
@@ -24,3 +24,4 @@ dj_local_conf.json
 log*.txt
 scratch/
 scratch*.py
+**/*.nfs*
```

Comment:
what is this ignore for? just curious

Reply:
Not sure why but sciviz generates these process files when it's running, e.g, `aeon_mecha/aeon/dj_pipeline/sciviz/.nfs000000000178f2a400000056`